server/lib/live-blogs/transform-live-blogs.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const cheerioTransform = require('../cheeerio-transform');
```

Comment:
doh, this file isn't even being used anywhere. which suggests live blogs are fine going through the rest of the pipeline? i'll add tests